static/icon-link.svg

```diff
@@ -0,0 +1 @@
+'data:image/svg+xml;charset=UTF-8,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" role="img"><path fill="#242121" d="M9.428,5.069a1.051,1.051,0,0,1,.74.98.991.991,0,0,1-1,1,1.167,1.167,0,0,1-.62-.22,2.614,2.614,0,0,0-.939-.2,2.585,2.585,0,0,0-2.062.88l-3,3a2.046,2.046,0,0,0,0,2.881,2.124,2.124,0,0,0,2.881,0,1.112,1.112,0,0,1,.84-.48.99.99,0,0,1,1,1,1.131,1.131,0,0,1-.46.861,4.11,4.11,0,0,1-1.5.939A3.978,3.978,0,0,1,1.165,9.15l3-3a4.83,4.83,0,0,1,3.561-1.52A4.827,4.827,0,0,1,9.428,5.069Zm3.44-5a3.93,3.93,0,0,1,1.94,1.08,3.96,3.96,0,0,1,0,5.621l-3,3A4.873,4.873,0,0,1,8.167,11.33a3.278,3.278,0,0,1-1.681-.5,1.062,1.062,0,0,1-.52-.881,1.007,1.007,0,0,1,1-1.02,1.185,1.185,0,0,1,.521.141,1.713,1.713,0,0,0,.82.26,2.594,2.594,0,0,0,2.12-.94l3-3a2.047,2.047,0,0,0,0-2.88,2.123,2.123,0,0,0-2.881,0,1.151,1.151,0,0,1-.86.48.99.99,0,0,1-1-1,1.151,1.151,0,0,1,.479-.86A3.639,3.639,0,0,1,11.728.008,4.369,4.369,0,0,1,12.868.068Z"/></svg>'
```

Comment:
...just realized this will require a change to the archetype.

I discovered that `markdown-it-toc-and-anchor` will escape any inline SVG I pass to `anchorSymbol` (whomp whomp) and that the postcss import is already replacing the url path with the html (hence the weird data uri string). I think `postcss-inline-svg` is the best way to go.